web/index.html

```diff
@@ -54,6 +54,7 @@
   // Initialize Firebase
   firebase.initializeApp(firebaseConfig);
   firebase.analytics();
+  firebase.auth().setPersistence(firebase.auth.Auth.Persistence.SESSION);
```

Comment:
Maybe add a small comment to say why this is necessary.

Reply:
Add comment:
https://github.com/student-hub/acs-upb-mobile/pull/250/commits/8c0e90bd1db807c70ea5422527345d766711da4c#diff-2804fa2a5ae6726cac5690433203db74702f6ee38877cb902a9d5095c419b578R187-R191

Reply:
I said why it's necessary, I can tell from the code that it's enabling "auth persistence", I just don't get what it is and why we need it.

Reply:
Here is the discussion about this issue:
https://github.com/student-hub/acs-upb-mobile/issues/207

Reply:
Maybe add a link to the issue in the comment?

Reply:
In other projects that are managed from Jira I saw using ticket names as reference, but considering that GitHub uses issue number I'll add a full link to related issue.